pkg/gadget-collection/gadgets/advise/seccomp/gadget.go

```diff
@@ -532,7 +532,7 @@ func (t *Trace) Generate(trace *gadgetv1alpha1.Trace) {
 
 	switch trace.Spec.OutputMode {
 	case gadgetv1alpha1.TraceOutputModeStatus:
-		policy := syscallArrToLinuxSeccomp(b)
+		policy := seccomptracer.SyscallArrToLinuxSeccomp(b)
```

Comment:
I was thinking that the tracer should return the syscalls names directly. What do you think?

Reply:
Yep, that makes sense. Let me try.